### DISCUSSION TITLE
article: Announcing: Theme Builder for Angular Material | Angular Material Dev

### DISCUSSION THREAD

**Discussion by giscus** [Angular-Material-Dev/community#8] [Article and Course Discussions]
# article: Announcing: Theme Builder for Angular Material | Angular Material Dev

Learn Material Design in Angular, Theming Angular Material Components and Developing your Angular application with Material Design like a pro.

https://angular-material.dev/articles/angular-material-theme-builder

**Comment by JGSolutions** [comment]
not free to get code

**Reply by shhdharmen** [reply]
Yeah, code is not free. But usage is free. You can copy the CSS variables freely and use it in your project.

**Comment by Klem35** [comment]
Thanks for the article. When using your method it seems to work but only primary color can be used. Accent and warn colors are not working.

**Reply by shhdharmen** [reply]
Support for accent and warn will anyways be going to deprecated. Read more at below:

1. https://material.angular.io/guide/theming#using-component-color-variants
2. https://material.angular.io/guide/material-2-theming#optional-add-backwards-compatibility-styles-for-color-variants

**Comment by kctang** [comment]
any plans to support customising typography?

**Reply by shhdharmen** [reply]
yes, soon!

**Reply by shhdharmen** [reply]
Hey, support for typography is available now at https://themes.angular-material.dev/typography

**Comment by mouragoncalves** [comment]
Video is not available.

**Comment by manishsg** [comment]
Hi 
Great product you build. I did a purchase and would recommend to others also. 

Please provide github link and also documentation or an article on installation and setup. Thanks !

**Reply by manishsg** [reply]
Works great. !

**Reply by shhdharmen** [reply]
Setup instructions are already given in the private GitHub repo's readme file.

**Comment by tagazok** [comment]
Very cool product!
But it does not seem to work with the last version of Material (v19).
Any chance to have it work?

**Reply by shhdharmen** [reply]
It will soon work with Angular 19! Stay tuned.

**Reply by shhdharmen** [reply]
Hey, it now supports Angular Material version 19 and 20, just pick your version when you export.
